autode/values.py

```diff
@@ -0,0 +1,203 @@
+import numpy as np
+from abc import ABC, abstractmethod
+from typing import Union
+from copy import deepcopy
+from autode.units import (Unit,
+                          ha, kjmol, kcalmol, ev,
+                          ang, a0, nm, pm, m,
+                          rad, deg)
+
+
+class Value(ABC):
+    """
+    Abstract base class for a value with a defined set of units, along with
+    perhaps other attributes and methods
+
+    x = Value(0.0)
+    """
+    implemented_units = []
+
+    def __repr__(self):
+        return self.__str__()
+
+    @abstractmethod
+    def __str__(self):
+        """String representation of this value"""
+
+    def copy(self):
+        return deepcopy(self)
+
+    def _convert_to_float(self, other):
```

Comment:
This function doesn't do what it says it does

Reply:
you make a good point

Reply:
better?